src/renderer.js

```diff
@@ -0,0 +1,111 @@
+import TVShowApp from './App.js';
+import commentModal from './modules/commentModal.js';
+import reservationModal from './modules/reservationModal.js';
+export default class Renderer{
+
+    static displayShows (shows) {
+      const movieList = document.querySelector('.show-list');
+      movieList.innerHTML = "";
+
+      for(let i = 0;i < shows.length;i++) {
+        movieList.innerHTML += `<li class='show-item'>
+            <figure>
+                <img src='${shows[i].image}'>
+                <figcaption>${shows[i].title}
+                  <img class='likebutton' src='https://img.icons8.com/ios-filled/50/000000/hearts.png'>
+                  <span id='likes_${shows[i].id}'>${shows[i].likes} ${shows[i].likes <= 1 ? 'Like':'Likes'}</span>
+                </figcaption>
+                <button id='comment_${shows[i].id}' class='btn btn-primary btn-md card-comment-btn comments'>Comments</button>
+                <button id='reservation_${shows[i].id}' class='btn btn-primary btn-md card-comment-btn reservations'>Reservations</button>
+            </figure>
+        </li>`
+      }
+
+      document.querySelector('.show-count').innerHTML = `Number of Shows: ${shows.length}`;
```

Comment:
- [ ] Hi @ibrahim777764, awesome job on this project. As requested from the requirements, we are expected to still create a specific function to calculate the counter numbers on each page even if the API provides that information. That should go the same with comments and reservations. We need a function though your counters are working perfectly :smiley: